tests/test-cases.js

```diff
@@ -1319,5 +1319,20 @@ var CUSTOM_TESTS = [
 				}
 			});
 		}
-	}
+	},
+	{
+		description: "Errors give correct row numbers with several chunks being processed",
+		expected: '[{"type":"FieldMismatch","code":"TooFewFields","message":"Too few fields: expected 12 fields but parsed 11","row":21},{"type":"FieldMismatch","code":"TooFewFields","message":"Too few fields: expected 12 fields but parsed 10","row":23}]',
```

Comment:
Well yes, the errors are in line 23 and 25, but considering 0-base of arrays and the headers you get the resulting rows of 21 and 23. Without the headers your assumption would be correct.